tests/plugins_test.go

```diff
@@ -143,9 +143,13 @@ func createBufGenYaml(t *testing.T, basedir string, plugin *plugin.Plugin) error
 	defer func() {
 		require.NoError(t, bufGenYaml.Close())
 	}()
+	opts := plugin.ExternalConfig.Registry.Opts
+	if len(opts) == 0 {
+		opts = plugin.DefaultOpts
```

Comment:
This is only needed for validate v0.6.13 plugins.

Reply:
Let's remove `v0.6.13` entirely and start publishing from `v0.9.0` onwards

Reply:
Done (although we'll need a BSR migration to remove existing published plugins).